packages/office-ui-fabric-react/src/components/MessageBar/docs/new/MessageBarBestPractices.md

```diff
@@ -0,0 +1,20 @@
+### Layout
+
+- A MessageBar is most commonly found near the top of an app, underneath the app's main command bar. For example, the MessageBar in Office is positioned beneath the ribbon, but above the document canvas.
+- Multiple MessageBars can appear at a time, but a given scenario or related set of scenarios should aim to only show one MessageBar at a time. MessageBars are rarely - shown in direct response to a user action; rather, they should be shown when there’s something the user should know about the overall app or document.
+- Place the message bar near the top of the relevant view, in a highly visible but unobtrusive location.
+- Use the right variant for the message type and urgency.
+- Don’t use buttons when a subtler link will suffice. Reserve the usage of button for when the MessageBar has a single ”hero” action that is useful at that particular moment. Using more than one button is discouraged.
+
+### Content
+
+- Be succinct and people are more likely to read everything you say.
+- Use one of the four icon options to indicate the type:
+  - a blue “i” for informational messages,
+  - a yellow shield for security-related information,
+  - a yellow warning triangle for non-blocking errors,
+  - a red “x” for critical errors.
+- **Title**: Limit yourself to 50 characters (including spaces) when creating titles to leave room for translation. Scanning the title should inform people about the general purpose of the message. Make sure to capitalize only the first word and any proper nouns.
```

Comment:
I'm a bit confused--the current MessageBar doesn't have a title and body text, just body text.